htdocs/js/jsx/incompletes_candidates_panel.jsx

```diff
@@ -0,0 +1,380 @@
+/*The config for the pagination of tables */
+var Config = React.createClass({
+    render: function() {
+        return <div>
+            <h2>Config</h2>
+            <label htmlFor="pageSize">Page Size:</label>
+            <select id="pageSize" value={this.props.pageSize} onChange={this.props.handlePageSizeChange}>
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                <option value="4">4</option>
+            </select>
```

Comment:
Deleted, this was used to change the amount of rows rendered via a select option but is not needed anymore.